player.py

```diff
@@ -0,0 +1,86 @@
+from discord.ext import commands
+from discord_slash import cog_ext, SlashContext
+from discord import FFmpegOpusAudio
+
+
+class Player(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(
+        brief="Radjo zaczyna grać.",
+        help="Bot dołącza do kanału głosowego, na którym jest użytkownik i odtwarza to co jest obecnie grane w radiu.",
+        pass_context=True
+    )
+    async def play(self, ctx: commands.Context):

```

Comment:
```suggestion
    async def play(self, ctx: commands.Context) -> None:
```
Skoro już część metody otypowana, to można całość